src/main/java/io/dockstore/EntryCreatorExample.java

```diff
@@ -46,14 +50,102 @@ public static void main(String[] args) throws ApiException, IOException {
         System.out.println(file);
 
         // add some metadata
+        returnDeposit.getMetadata().setPublicationDate("2025-03-03");
         returnDeposit.getMetadata().setTitle("foobar title");
         returnDeposit.getMetadata().setUploadType(DepositMetadata.UploadTypeEnum.POSTER);
-        returnDeposit.getMetadata().setDescription("This is a first upload");
+        // just above https://developers.zenodo.org/#list
+        // there's a note that says " For string fields that allow HTML (e.g. description, notes), for security reasons, only the following tags are accepted: a, abbr, acronym, b, blockquote, br, code, caption, div, em, i, li, ol, p, pre, span, strike, strong, sub, table, caption, tbody, thead, th, td, tr, u, ul. "
```

Comment:
What does Zenodo do if the submitted HTML contains verbotten tags?  Filter them out?  Reject the submission?

Reply:
In my experiments, they just don't work (i.e. tag ignored). 

Bold (typo) and marquee ignored
![Screenshot from 2025-04-03 15-48-20](https://github.com/user-attachments/assets/003d4a2d-f678-4183-aacf-ae623fa5dbb7)

Bold (typo) ignored (strike-through ok)
![Screenshot from 2025-04-03 15-48-47](https://github.com/user-attachments/assets/4115c4f9-d3b1-4d85-9a5f-9060fc24dde8)